commercial/app/model/merchandise/events/Eventbrite.scala

```diff
@@ -130,22 +130,32 @@ object Eventbrite extends ExecutionContexts with Logging {
   trait TicketHandler {
     def tickets: Seq[Ticket]
 
-    val displayPriceRange = {
+    lazy val displayPriceRange: Option[String] = {
```

Comment:
Ah yes! Good spot. I originally did that in a brief spot of laziness before immediately abandoning the idea. My idea was to make it lazy and filter out all Events that have no tickets, but this was a terrible idea for many reason - forgot to delete the `lazy` though.